roles/nginx/templates/h5bp/directive-only/ssl.conf

```diff
@@ -25,7 +21,7 @@ ssl_session_timeout  24h;
 #
 # Note that you'll have to define and rotate the keys securely by yourself. In absence
 # of such infrastructure, consider turning off session tickets:
-#ssl_session_tickets off;
+ssl_session_tickets off;
```

Comment:
:+1: this is a good default.

https://github.com/mozilla/server-side-tls/issues/135